src/cpp/include/openvino/genai/visual_language/pipeline.hpp

```diff
@@ -97,7 +97,25 @@ class OPENVINO_GENAI_EXPORTS VLMPipeline {
     /// To disable it for non-chat mode, please, use custom_chat_template eq "" or set generation_config.apply_chat_template to false.
     VLMDecodedResults generate(
         const std::string& prompt,
-        const std::vector<ov::Tensor>& rgbs,
+        const std::vector<ov::Tensor>& images,
+        const GenerationConfig& generation_config,
+        const StreamerVariant& streamer
+    );
+
+    /// @brief Generate a response given a prompt and any number of
+    /// uint8 RGB images with [NHWC] or [HWC] layout.
```

Comment:
Please update this description and add information about video.

Reply:
Done.